From dc91e1b7a3218a0a186dfd260d928f2c9ebd334d Mon Sep 17 00:00:00 2001
From: Stephane Danieau <stephane.danieau@st.com>
Date: Tue, 5 Dec 2023 11:23:29 +0100
Subject: [PATCH 0654/1141] arm64: defconfig: Add Cirrus CS42L51 audio codec
 support

Add CONFIG_SND_SOC_CS42L51 to enable stm32mp257f-dk audio codec support

Change-Id: I0d7377da1af25a58db6bfbe5fbb9ab46cd7b291a
Signed-off-by: Stephane Danieau <stephane.danieau@st.com>
---
 arch/arm64/configs/fragment-02-defconfig-addons.config | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/configs/fragment-02-defconfig-addons.config b/arch/arm64/configs/fragment-02-defconfig-addons.config
index 6a059e3f6b4a..e0ab86d31c00 100644
--- a/arch/arm64/configs/fragment-02-defconfig-addons.config
+++ b/arch/arm64/configs/fragment-02-defconfig-addons.config
@@ -72,6 +72,8 @@ CONFIG_RTC_DRV_STM32=y
 CONFIG_SENSORS_MR75203=y
 CONFIG_SERIAL_STM32=y
 CONFIG_SERIAL_STM32_CONSOLE=y
+CONFIG_SND_SOC_CS42L51=m
+CONFIG_SND_SOC_CS42L51_I2C=m
 CONFIG_SND_SOC_STM32_I2S=m
 CONFIG_SND_SOC_STM32_MDF=m
 CONFIG_SND_SOC_STM32_SAI=m
-- 
2.39.2

